data/symbolic_prefixes.tsv

```diff
@@ -0,0 +1,7 @@
+symbol	name	can_be_disease_or_phenotype	description
```

Comment:
@sabrinatoro (@nicolevasilevsky IDK if this is something you are reviewing / have time to review), @matentzn 
I'm working on: 
- https://github.com/monarch-initiative/mondo-ingest/issues/110

The prefix symbols OMIM has make it a lot clearer what is / isn't a phenotype. Note that they don't use the word 'disease'. I want a curator / someone more knowledgeable to review this though. I went over the [symbol explanations](https://www.omim.org/help/faq#1_3), and from what I could tell, the only symbol that can never be a phenotype/disease is prefixed by an `*`:
> An asterisk (*) before an entry number indicates a gene.

It looks like all the other ones could potentially be a disease/phenotype.

The only other one I was considering adding to exclusions was `^`:
> A caret (^) before an entry number means the entry no longer exists because it was removed from the database or moved to another entry as indicated.

However, these are excluded in other ways. I don't want to use the tag `MONDO:excludeNonDisease` on these.